diff --unified --recursive --text package.orig/diyHue-2022.8/BridgeEmulator/configManager/configHandler.py package.new/diyHue-2022.8/BridgeEmulator/configManager/configHandler.py
--- package.orig/diyHue-2022.8/BridgeEmulator/configManager/configHandler.py	2023-01-20 17:19:26.077471868 +0100
+++ package.new/diyHue-2022.8/BridgeEmulator/configManager/configHandler.py	2023-01-20 17:23:16.659539895 +0100
@@ -52,8 +52,6 @@
                 # updgrade config
                 if "homeassistant" not in config:
                     config["homeassistant"] = {"enabled": False}
-                if "yeelight" not in config:
-                    config["yeelight"] = {"enabled": True}
                 if "native_multi" not in config:
                     config["native_multi"] = {"enabled": True}
                 if "tasmota" not in config:
@@ -66,8 +64,6 @@
                     config["esphome"] = {"enabled": True}
                 if "hyperion" not in config:
                     config["hyperion"] = {"enabled": True}
-                if "tpkasa" not in config:
-                    config["tpkasa"] = {"enabled": True}
 
                 if int(config["swversion"]) < 1952154030:
                     config["swversion"] = "1952086020"
@@ -94,14 +90,12 @@
                     "hue": {},
                     "tradfri": {},
                     "homeassistant": {"enabled":False},
-                    "yeelight": {"enabled":True},
                     "native_multi": {"enabled":True},
                     "tasmota": {"enabled":True},
                     "wled": {"enabled":True},
                     "shelly": {"enabled":True},
                     "esphome": {"enabled":True},
                     "hyperion": {"enabled":True},
-                    "tpkasa": {"enabled":True},
                 }
             # load lights
             if os.path.exists(self.configDir + "/lights.yaml"):
diff --unified --recursive --text package.orig/diyHue-2022.8/BridgeEmulator/lights/discover.py package.new/diyHue-2022.8/BridgeEmulator/lights/discover.py
--- package.orig/diyHue-2022.8/BridgeEmulator/lights/discover.py	2023-01-20 17:19:26.160804744 +0100
+++ package.new/diyHue-2022.8/BridgeEmulator/lights/discover.py	2023-01-20 17:22:16.606536099 +0100
@@ -5,7 +5,7 @@
 import uuid
 from time import sleep
 from datetime import datetime
-from lights.protocols import tpkasa, wled, mqtt, hyperion, yeelight, hue, deconz, native, native_single, native_multi, tasmota, shelly, esphome, tradfri
+from lights.protocols import wled, mqtt, hyperion, hue, deconz, native, native_single, native_multi, tasmota, shelly, esphome, tradfri
 from services.homeAssistantWS import discover
 import HueObjects
 from functions.core import nextFreeId
@@ -114,8 +114,6 @@
         deconz.discover(detectedLights, bridgeConfig["config"]["deconz"])
     if bridgeConfig["config"]["homeassistant"]["enabled"]:
         discover(detectedLights)
-    if bridgeConfig["config"]["yeelight"]["enabled"]:
-        yeelight.discover(detectedLights)
     # native_multi probe all esp8266 lights with firmware from diyhue repo
     if bridgeConfig["config"]["native_multi"]["enabled"]:
         native_multi.discover(detectedLights, device_ips)
@@ -133,8 +131,6 @@
     tradfri.discover(detectedLights, bridgeConfig["config"]["tradfri"])
     if bridgeConfig["config"]["hyperion"]["enabled"]:
         hyperion.discover(detectedLights)
-    if bridgeConfig["config"]["tpkasa"]["enabled"]:
-        tpkasa.discover(detectedLights)
     bridgeConfig["temp"]["scanResult"]["lastscan"] = datetime.now().strftime(
         "%Y-%m-%dT%H:%M:%S")
     for light in detectedLights:
@@ -148,7 +144,7 @@
                         lightObj.protocol_cfg["ip"] = light["protocol_cfg"]["ip"]
                         lightIsNew = False
                         break
-                elif light["protocol"] in ["yeelight", "tasmota", "tradfri", "hyperion", "tpkasa"]:
+                elif light["protocol"] in ["tasmota", "tradfri", "hyperion"]:
                     if lightObj.protocol_cfg["id"] == light["protocol_cfg"]["id"]:
                         logging.info("Update IP for light " + light["name"])
                         lightObj.protocol_cfg["ip"] = light["protocol_cfg"]["ip"]
Only in package.orig/diyHue-2022.8/BridgeEmulator/lights/protocols: hue_bl.py
diff --unified --recursive --text package.orig/diyHue-2022.8/BridgeEmulator/lights/protocols/__init__.py package.new/diyHue-2022.8/BridgeEmulator/lights/protocols/__init__.py
--- package.orig/diyHue-2022.8/BridgeEmulator/lights/protocols/__init__.py	2023-01-20 17:19:26.160804744 +0100
+++ package.new/diyHue-2022.8/BridgeEmulator/lights/protocols/__init__.py	2023-01-20 17:20:56.296976793 +0100
@@ -1,3 +1,3 @@
-from lights.protocols import wled, hyperion, yeelight, tasmota, shelly, mi_box, hue, deconz, domoticz, tradfri, native, native_single, native_multi, esphome, mqtt, wiz, milight, homeassistant_ws, tpkasa, hue_bl
+from lights.protocols import wled, hyperion, tasmota, shelly, mi_box, hue, deconz, domoticz, tradfri, native, native_single, native_multi, esphome, mqtt, wiz, milight, homeassistant_ws
 
-protocols = [wled, hyperion, yeelight, tasmota, shelly, mi_box, hue, deconz, domoticz, tradfri, native, native_single, native_multi, esphome, mqtt, wiz, milight, homeassistant_ws, tpkasa, hue_bl]
+protocols = [wled, hyperion, tasmota, shelly, mi_box, hue, deconz, domoticz, tradfri, native, native_single, native_multi, esphome, mqtt, wiz, milight, homeassistant_ws]
Only in package.orig/diyHue-2022.8/BridgeEmulator/lights/protocols: tpkasa.py
Only in package.orig/diyHue-2022.8/BridgeEmulator/lights/protocols: yeelight.py
